### ISSUE TITLE
[Feature Request] Export Settings

### ISSUE THREAD

**Issue by Tekkiech** [w2sv/FileNavigator#77]
### Overview

Adding the ability to export the app's settings in order to transfer over to a new phone, for example.

**Comment by businessBoris** [comment]
Yes please, we'd love a backup option that backs up with a scheduler.
Update: manners!
